daemon/api.go

```diff
@@ -1232,7 +1234,7 @@ func postSnaps(c *Command, r *http.Request, user *auth.UserState) Response {
 
 	dangerousOK := isTrue(form, "dangerous")
 	devmode := isTrue(form, "devmode")
```

Comment:
can you nuke the `devmode` variable? it's a long function, and having one of the flags in a variable makes me wonder what it's being used for down below, and why the others aren't getting used. AFAICT in the review it isn't used at all, which is fine, but dropping it (like the others) makes reading the code easier